src/Compilers/CSharp/Portable/Symbols/Synthesized/GeneratedNames.cs

```diff
@@ -435,6 +435,12 @@ internal static string MakeIteratorCurrentThreadIdFieldName()
             return "<>l__initialThreadId";
         }
 
+        internal static string MakeEncStructExpansionFieldName()
+        {
+            Debug.Assert((char)GeneratedNameKind.EncStructExpansionField == 'v');
+            return "<>v_encStructExpansion";
```

Comment:
Thank you! It seems <>v__encStructExpansion is the right choice.

---
In reply to: [134861840](https://github.com/dotnet/roslyn/pull/21664#discussion_r134861840) [](ancestors = 134861840)